docs/source/nlp/question_answering.rst

```diff
@@ -0,0 +1,339 @@
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| optim.name                                | string          | adamw                                                                            | The optimizer to use for training                                                                            |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| optim.lr                                  | float           | 2e-5                                                                             | The learning rate to use for training                                                                        |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| optim.weight_decay                        | float           | 0.0                                                                              | The weight decay to use for training                                                                         |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.name                          | string          | SquareRootAnnealing                                                              | The warmup schedule                                                                                          |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.warmup_ratio                  | float           | 0.0                                                                              | The warmup ratio                                                                                             |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
```

Comment:
how about removing max_epochs, batch_size, shuffle and optim related params since they are not model specific?

Reply:
ok